auth/src/test/java/com/firebase/ui/auth/test_helpers/FacebookProviderShadow.java

```diff
@@ -44,8 +41,6 @@ public FacebookProviderShadow() {
         }
     }
 
-    public void __constructor__(Activity activity, List<String> scopes) {}
```

Comment:
Are we sure none of the tests are implicitly using this?  Android Studio would not be able to tell if this was unused.